proposals/save_ml_model.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "save_ml_model",
+    "summary": "Save a ML model",
+    "description": "Saves a machine learning model as part of a batch job.\n\nThe model will be accompanied by a separate STAC Item that implements the [ml-model extension](https://github.com/stac-extensions/ml-model).",
+    "categories": [
+        "machine learning",
+        "import"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The data to store as a machine learning model.",
+            "schema": {
+                "type": "object",
+                "subtype": "ml-model"
+            }
+        },
+        {
+            "name": "options",
+            "description": "Additional parameters to create the file(s).",
+            "schema": {
+                "type": "object",
+                "additionalParameters": false
+            },
+            "default": {},
+            "optional": true
+        }
+    ],
+    "returns": {
+        "description": "Returns `false` if the process failed to store the model, `true` otherwise.",
```

Comment:
IMHO this is a bad design pattern: if the storage failed, an error should be thrown, 
it's just too error prone to return a boolean nobody is going to check.

I think it's cleaner to return nothing (`null`) or return the original `data`

Reply:
Yeah, this was copied over from save_result. The same also applies there and the boolean was not the wisest thing, it should indeed throw an error and I assume most implementations will actually do that, right?

Reply:
Created a separate issue: https://github.com/Open-EO/openeo-processes/issues/334